config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
Just so I'm clear in my head about the bonus exercises, those that are `core:false, unlocked_by:null` are available as soon as `hello-world` has been submitted? Is that any different to having them be `unlocked_by: hello-world`?